src_graphql/github_domain.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
```

Comment:
@DubeySandeep are you okay with us using your code with this kind of copyright notice, or would you prefer that we explicitly acknowledge your original repo?

Reply:
@DubeySandeep a friendly reminder to reply here. Thanks!

Reply:
@U8NWXD @SD-13, sorry for missing it, can you please make sure to assign me to the PR wherever I need to take action? I'll make sure not to miss the PRs assigned to me.

I think this is a completely different code and you won't require any permissions.

Reply:
> One option is for us to use a copyright notice attributing the oppia authors (what we have now)

This should be fine!